lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -2724,6 +2741,10 @@ void FIRRTLLowering::addIfProceduralBlock(Value cond,
 ///
 FIRRTLLowering::UnloweredOpResult
 FIRRTLLowering::handleUnloweredOp(Operation *op) {
+  // Push nested operations onto the worklist such that they are lowered.
+  for (auto &region : op->getRegions())
+    addToWorklist(region);
```

Comment:
Can you put this under `if (!isa<FIRRTLDialect>(op->getDialect())) {` below and causes an error when there is unhandled FIRRTL operation with a region? I think it's more predictable not to silently lower unhandled FIRRTL operations with a region.

Reply:
Oh yeah that's a great idea! Let me change that.

Reply:
Done 👍